src/components/game-table.jsx

```diff
@@ -44,6 +44,7 @@ module.exports = React.createClass({
           <th>Player</th>
           <th className="hide_sm">League</th>
           <th>Score</th>
+          <th>Streak</th>
```

Comment:
`<th  className="hide_sm">Streak</th>` just keeps the mobile version easy to use..